NeoMathEngine/src/GPU/CUDA/CudaMathEngineDnnDropout.cu

```diff
@@ -68,9 +67,9 @@ void CCudaMathEngine::Dropout( const CDropoutDesc& dropoutDesc,
 		dim3 blockCount;
 		dim3 threadCount;
 
-		getCudaTaskGrid2D(blockCount, threadCount, batchLength, (maskSize + 3) / 4);
+		getCudaTaskGrid2D(blockCount, threadCount, batchLength, (maskSize + desc.CacheSize - 1) / desc.MaskAlign);
```

Comment:
Wrong using `desc.CacheSize`, should be `desc.MaskAlign`

Reply:
ooops